index.html

```diff
@@ -45,143 +45,201 @@ <h3>
           Tomer Sedan
         </h3>
         <hr>
-        <a class="link" href="https://github.com/Sordamente">GitHub</a>
+        <a href="https://github.com/Sordamente">
+          <img src="./images/github.svg" alt="github-logo">
+          GitHub
+        </a>
```

Comment:
What icon will be used for personal websites? Will we link to personal websites?

Reply:
Hmm, maybe the grey globe icon which is the default for websites?

Reply:
Yeah that should suffice

Reply:
Yup that sounds good 👍🏼